Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,287 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+
+#I need help in coding here. If this doesn't get removed. If someone knows the commands or how to do it, I'd appreciate it if you tell me. Assuming this topic isn't removed by then#
```

Comment:
You might want to consider removing the `I need help in coding here` comment, it's no longer needed. Same thing could apply for the spoiler warning.

Reply:
done. but the spoiler warning stays. someone may want to watch it but doesn't wanna get spoiled